docs/install/helm-install/rancher.md

```diff
@@ -7,7 +7,7 @@ tags:
 
 # Install Gluu Server Using Rancher Marketplace
 
-For a more generic `Gluu Flex` installation on Rancher, you can follow [this](https://docs.gluu.org/vreplace-flex-version/admin/recipes/getting-started-rancher/) comprehensive guide.
+For a more generic `Gluu Flex` installation on Rancher, you can follow [this](https://docs.gluu.org/head/admin/recipes/getting-started-rancher/) comprehensive guide.
```

Comment:
head should reference `vreplace-flex-version` so it gets replaced on official releases

Reply:
The only reason I changed to `head`, is that if the user is at `https://docs.gluu.org/head/install/helm-install/rancher/` and clicked on the hyperlink, he'll be redirected to `https://docs.gluu.org/vreplace-flex-version/admin/recipes/getting-started-rancher/` and gets `Error 404`

Reply:
Thats fine because head is meant to be the bleeding edge version. Users would use nightly for the latest stable bleeding edge